Chapter03/packing_1.py

```diff
@@ -4,14 +4,16 @@
 """
 
 
-USERS = [(i, f"first_name_{i}", "last_name_{i}") for i in range(1_000)]
+USERS = [(i, f"first_name_{i}", f"last_name_{i}") for i in range(1_000)]
 
 
 class User:
     def __init__(self, user_id, first_name, last_name):
         self.user_id = user_id
         self.first_name = first_name
         self.last_name = last_name
+    def __repr__(self):
```

Comment:
Could you please add a new line before the ``__repr__`` to comply with PEP-8?

With respect to the method, ``repr`` should call in turn the ``repr`` of the attributes, and it would be good to use string formatting. See what I mean :)

```suggestion
class User:
    def __init__(self, user_id, first_name, last_name):
        self.user_id = user_id
        self.first_name = first_name
        self.last_name = last_name

    def __repr__(self):
        return f"self.__class__.__name__({self.user_id!r}, {self.first_name!r}, {self.last_name!r})"
```

This complies with the general convention that ``eval(repr(x))`` should be ``x``.

In this case if you create something like
```python
user1 = User(1, "name_1", "last_name_1")
```

Its ``repr`` would give
```python
>>> repr(user1)
User(1, 'name_1', 'last_name_1')
```

Thanks for spotting this and for your contribution!

Reply:
Fix USERS f-string last name initialization error, (@rmariano) complies with eval(repr(x)) should be x conviction. 
Add repr for User Class on printing behalf, (@rmariano) comply pep-8.
Thanks for previous remarks clearing code in python.